forge-gui/res/cardsfolder/upcoming/serum_snare.txt

```diff
@@ -0,0 +1,10 @@
+Name:Serum Snare
+ManaCost:1 U
+Types:Instant
+A:SP$ ChangeZone | Origin$ Battlefield | Destination$ Hand | ValidTgts$ Permanent.nonLand | TgtPrompt$ Select target nonland permanent | RememberLKI$ True | SubAbility$ DBProliferate | SpellDescription$ Return target nonland permanent to its owner's hand. If that permanent had mana value 3 or less, proliferate.
+SVar:DBProliferate:DB$ Proliferate | ConditionCheckSVar$ X | ConditionSVarCompare$ LE3 | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:X:Targeted$CardManaCostLKI
```

Comment:
- You aren't even checking the Remembered

- You should also write it without SVar

Reply:
Like I said avoid the SVar -> ConditionDefined$

Reply:
Still incorrect

Present is valid syntax only, so use `Card.cmcLE3` and get ride of the Compare$ because of defaults